src/plugins/console/public/application/components/help_popover.tsx

```diff
@@ -0,0 +1,133 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import React from 'react';
+import { i18n } from '@kbn/i18n';
+import {
+  EuiPopover,
+  EuiTitle,
+  EuiText,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiButtonIcon,
+  EuiSpacer,
+} from '@elastic/eui';
+import { useServicesContext } from '../contexts';
+
+interface HelpPopoverProps {
+  button: any;
+  isOpen: boolean;
+  closePopover: () => void;
+  resetTour: () => void;
+}
+
+export const HelpPopover = ({ button, isOpen, closePopover, resetTour }: HelpPopoverProps) => {
+  const { docLinks } = useServicesContext();
+
+  return (
+    <EuiPopover
+      button={button}
+      isOpen={isOpen}
+      closePopover={closePopover}
+      anchorPosition="downRight"
+      buffer={4}
+      data-test-subj="consoleHelpPopover"
+    >
+      <EuiTitle size="xs">
+        <h4>
+          {i18n.translate('console.helpPopover.title', {
+            defaultMessage: 'Elastic Console',
+          })}
+        </h4>
+      </EuiTitle>
+
+      <EuiSpacer size="s" />
+
+      <EuiText style={{ width: 300 }} color="subdued" size="s">
+        <p>
+          {i18n.translate('console.helpPopover.description', {
+            defaultMessage:
+              'Our UI for interacting with Elasticsearch clusters using QueryDSL. Easily run queries, manage settings, and view search results.',
+          })}
+        </p>
+      </EuiText>
+
+      <EuiSpacer />
+
+      <EuiFlexGroup gutterSize="m" direction="column">
+        <EuiFlexItem>
+          <EuiFlexGroup justifyContent="spaceBetween" alignItems="center">
+            <EuiFlexItem grow={false}>
+              <p>
+                {i18n.translate('console.helpPopover.aboutConsoleLabel', {
+                  defaultMessage: 'About Console',
+                })}
+              </p>
+            </EuiFlexItem>
+            <EuiFlexItem grow={false}>
+              <EuiButtonIcon
+                iconType="popout"
+                href={docLinks.console.guide}
+                target="_blank"
+                color="text"
+                aria-label={i18n.translate('console.helpPopover.aboutConsoleButtonAriaLabel', {
+                  defaultMessage: 'About Console link',
+                })}
+              />
+            </EuiFlexItem>
+          </EuiFlexGroup>
+        </EuiFlexItem>
+
+        <EuiFlexItem>
+          <EuiFlexGroup justifyContent="spaceBetween" alignItems="center">
+            <EuiFlexItem grow={false}>
+              <p>
+                {i18n.translate('console.helpPopover.aboutQueryDSLLabel', {
+                  defaultMessage: 'About QueryDSL',
```

Comment:
```suggestion
                  defaultMessage: 'About Query DSL',
```

Reply:
Fixed in [3c7b985](https://github.com/elastic/kibana/pull/191092/commits/3c7b985922c7d10079d829e6157328c78da85fed).